.github/workflows/build-wheel-linux-arm64.yaml

```diff
@@ -105,26 +105,28 @@ jobs:
         path: mlir/Enzyme
 
     # Cache external project builds
```

Comment:
```suggestion
    # Restore external project builds
```

Reply:
Well, this will do both, I think cache is appropriate here.